ui/src/components/ConfigurationForm/ConfigurationForm.consts.ts

```diff
@@ -36,6 +36,10 @@ export const EFFECT_TYPE_OPTIONS: FormSelectOption<EffectType>[] = [
     label: 'IO (cats-effect)',
     value: EffectType.IO,
   },
+  {
+    label: 'Sync (direct-style)',
```

Comment:
maybe let's label this as `No effect (direct style)`? As it's a dropdown value for "Effect type". 

Or maybe it would be best to rename the whole dropdown. Maybe "Stack"? With values: `Functional (IO, cats-effect)`, `Functional (ZIO)`, `Direct-style (Ox)`, `Future`? WDYT?

Reply:
I like the Stack idea. This would mean quite a refactor to make the code reflect it accurately in all layers (StackType instead of EffectType, etc.), so maybe in the next PR?

Reply:
Ah, I was just thinking of changing the labels, but renaming in code is probably the right thing to do ;)